formtools/utils.py

```diff
@@ -1,9 +1,6 @@
-from __future__ import unicode_literals
-
 # Do not try cPickle here (see #18340)
```

Comment:
This comment can go too I think, there's no such thing as `cPickle` on Python 3!